python/lsst/jointcal/ccdImage.cc

```diff
@@ -66,13 +65,11 @@ void declareCcdImageList(py::module &mod) {
 
     cls.def("sizeValidForFit", &CcdImageList::sizeValidForFit);
 
-    // Make this behave like a std::list, which it is, but we can't wrap children of it easily.
+    // Make this behave like a std::list (which it is but we can't wrap std::list subclasses trivially)
```

Comment:
Do we want to try to sort out some of DM-4043 now (what it would take to decide that question), or should we just try to do that in the next sprint?

`ccdImageList` can probably be turned into a List without too much trouble, but the `*StarList` are all built on `CountedRefs`, which are `boost::intrusive_ptr`, so I fear more complication there.